vendor-extra/ContentPageBundle/tests/ViewConvertingTestCase.php

```diff
@@ -8,14 +8,33 @@
 use Libero\ViewsBundle\Views\CallbackViewConverter;
 use Libero\ViewsBundle\Views\View;
 use Libero\ViewsBundle\Views\ViewConverter;
+use LogicException;
 
 trait ViewConvertingTestCase
 {
-    final protected function createConverter() : ViewConverter
+    final protected function createConverter(bool $includeTemplate = false) : ViewConverter
```

Comment:
boolean argument (especially without keyword-based arguments like in Python) are not expressive, consider `$this->createConverter(self::INCLUDE_TEMPLATE)` which can still be a boolean but expressed as a constant. Very low cost for readability

Reply:
Agreed. Since traits can't have constants, just going to remove the option.